nautobot/tenancy/api/serializers.py

```diff
@@ -33,7 +33,7 @@ class Meta:
 
 class TenantSerializer(NautobotModelSerializer, TaggedModelSerializerMixin):
     url = serializers.HyperlinkedIdentityField(view_name="tenancy-api:tenant-detail")
-    group = NestedTenantGroupSerializer(required=False)
+    tenant_group = NestedTenantGroupSerializer(required=False)
```

Comment:
This needs to be documented as a renamed serializer field in the upgrading documentation.